### PR TITLE
Update potential-pitfalls-with-plinq.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: docs/standard/parallel-programming/potential-pitfalls-with-plinq.md [hunk 1]
```diff
@@ -10,26 +10,26 @@ helpviewer_keywords:
...
 
-## Avoid Over-Parallelization
+## Avoid over-parallelization
 
-By using the `AsParallel` operator, you incur the overhead costs of partitioning the source collection and synchronizing the worker threads. The benefits of parallelization are further limited by the number of processors on the computer. There is no speedup to be gained by running multiple compute-bound threads on just one processor. Therefore, you must be careful not to over-parallelize a query.
+By using the `AsParallel` method, you incur the overhead costs of partitioning the source collection and synchronizing the worker threads. The benefits of parallelization are further limited by the number of processors on the computer. There is no speedup to be gained by running multiple compute-bound threads on just one processor. Therefore, you must be careful not to over-parallelize a query.
```
Comment (Youssef1313) [inline docs/standard/parallel-programming/potential-pitfalls-with-plinq.md]: I believe that `AsParallel` is a method.  It's not accurate to call it as operator.

File: docs/standard/parallel-programming/potential-pitfalls-with-plinq.md [hunk 2]
```diff
@@ -55,32 +55,30 @@ FileStream fs = File.OpenWrite(...);
...
 
-## Prefer ForAll to ForEach When It Is Possible
+## Prefer ForAll to ForEach when it is possible
 
 Although PLINQ executes a query on multiple threads, if you consume the results in a `foreach` loop (`For Each` in Visual Basic), then the query results must be merged back into one thread and accessed serially by the enumerator. In some cases, this is unavoidable; however, whenever possible, use the `ForAll` method to enable each thread to output its own results, for example, by writing to a thread-safe collection such as <xref:System.Collections.Concurrent.ConcurrentBag%601?displayProperty=nameWithType>.
 
-The same issue applies to <xref:System.Threading.Tasks.Parallel.ForEach%2A?displayProperty=nameWithType> In other words, `source.AsParallel().Where().ForAll(...)` should be strongly preferred to
+The same issue applies to <xref:System.Threading.Tasks.Parallel.ForEach%2A?displayProperty=nameWithType>. In other words, `source.AsParallel().Where().ForAll(...)` should be strongly preferred to `Parallel.ForEach(source.AsParallel().Where(), ...)`.
```
Comment (Youssef1313) [inline docs/standard/parallel-programming/potential-pitfalls-with-plinq.md]: I've added a missing period before "`In other words`", and removed the line break between "should be strongly preferred to" and "`Parallel.ForEach(......`"

File: docs/standard/parallel-programming/potential-pitfalls-with-plinq.md [hunk 3]
```diff
@@ -55,32 +55,30 @@ FileStream fs = File.OpenWrite(...);
...
 
-## Do Not Assume that Iterations of ForEach, For and ForAll Always Execute in Parallel
+## Don't assume that iterations of ForEach, For, and ForAll always execute in parallel
 
-It is important to keep in mind that individual iterations in a <xref:System.Threading.Tasks.Parallel.For%2A?displayProperty=nameWithType>, <xref:System.Threading.Tasks.Parallel.ForEach%2A?displayProperty=nameWithType> or <xref:System.Linq.ParallelEnumerable.ForAll%2A> loop may but do not have to execute in parallel. Therefore, you should avoid writing any code that depends for correctness on parallel execution of iterations or on the execution of iterations in any particular order.
+It is important to keep in mind that individual iterations in a <xref:System.Threading.Tasks.Parallel.For%2A?displayProperty=nameWithType>, <xref:System.Threading.Tasks.Parallel.ForEach%2A?displayProperty=nameWithType>, or <xref:System.Linq.ParallelEnumerable.ForAll%2A> loop may but do not have to execute in parallel. Therefore, you should avoid writing any code that depends for correctness on parallel execution of iterations or on the execution of iterations in any particular order.
```
Comment (Youssef1313) [inline docs/standard/parallel-programming/potential-pitfalls-with-plinq.md]: added a comma before "or"

**Review by BillWagner — APPROVED** [review]
These changes LGTM @Youssef1313 

I'll :shipit: now.

**Pull request merged**